test/lib/mock.js

```diff
@@ -76,7 +76,22 @@ module.exports = function mock(options) {
         });
         return s;
       });
+  }
 
+  function deleteAws() {
+    var bucket = config.get('img.uploads.dest.public');
+    var u = '/' + bucket + '?delete';
+    return nock('https://s3.amazonaws.com')
+      .post(u)
+      .reply(200, function(uri, body) {
+        var id = body.match(/<Key>([0-9a-f]{32})<\/Key>/)[1];
+        var s = through();
+        s.setEncoding = function() {};
+        local.delete(id).done(function() {
+          s.end();
+        });
+        return s;
+      });
```

Comment:
Yes, exactly. The `nock.reply` method expects either a String or a Stream returned. Since the local driver is still async, I'm returning an empty stream that finishes once the local driver is complete.